USAGE_desktop_D3D12.md

```diff
@@ -276,77 +277,37 @@ Optional arguments:
   --pbi-all             Print all block information.
   --pbis <index1,index2>Print block information between block index1 and block index2.
 
-Windows-only:
   --fwo <x,y>           Force windowed mode if not already, and allow setting of a custom window location.
                         (Same as --force-windowed-origin)
-
```

Comment:
Did you mean to delete this section and the "Windows-only" bit at line 279?

Reply:
Heh, sure is D3D12-specific.  Wow, that inapplicable documentation has been in there a long time.